app/assets/javascripts/teams.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function() {
+    $('#team-sources input[id^=team_sources_attributes]:radio').change(function() {
+        var helpText,
+            helpNode = $($(this).parents()[2]).find('.help-block');
```

Comment:
Same here, I'd suggest to use a data attribute as well - makes selection easier.